forge-gui/res/cardsfolder/v/vesuvan_duplimancy.txt

```diff
@@ -1,9 +1,8 @@
 Name:Vesuvan Duplimancy
 ManaCost:3 U
 Types:Enchantment
-T:Mode$ SpellCast | ValidCard$ Card.YouCtrl  | Execute$ TrigCopyTarget | ValidActivatingPlayer$ You | IsSingleTarget$ True | TriggerZones$ Battlefield | TargetsValid$ Creature.YouCtrl+inZoneBattlefield,Artifact.YouCtrl+inZoneBattlefield | RememberValidCards$ True | TriggerDescription$ Whenever you cast a spell that targets only a single artifact or creature you control, create a token that's a copy of that artifact or creature, except it's not legendary.
-SVar:TrigCopyTarget:DB$ CopyPermanent | Choices$ Permanent.IsRemembered | NonLegendary$ True | SubAbility$ DBCleanup 
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl  | Execute$ TrigCopyTarget | ValidActivatingPlayer$ You | IsSingleTarget$ True | TriggerZones$ Battlefield | TargetsValid$ Creature.YouCtrl+inZoneBattlefield,Artifact.YouCtrl+inZoneBattlefield | TriggerDescription$ Whenever you cast a spell that targets only a single artifact or creature you control, create a token that's a copy of that artifact or creature, except it's not legendary.
+SVar:TrigCopyTarget:DB$ CopyPermanent | Defined$ TriggeredSpellAbilityTarget | Choices$ Card | NonLegendary$ True
```

Comment:
This should already work with `TriggeredSpellAbilityTargetingCards`, so we can get rid of `SpellAbilityTarget` soon